.github/workflows/python-tests.yaml

```diff
@@ -143,7 +143,7 @@ jobs:
             PYTHON_VERSION=${{ matrix.python-version }}
             BASE_IMAGE=prefect-conda
             PREFECT_EXTRAS=[dev]
-          tags: prefecthq/prefect:dev-python${{ matrix.python-version }}-conda
+          tags: prefecthq/prefect-dev:sha-$(git rev-parse --short HEAD)-python${{ matrix.python-version }}-conda
```

Comment:
This doesn't work because it's being passed to the Docker build/push action. We'll need to generate this separately and template it in at the actions level?